.gitlab-ci.yml

```diff
@@ -7,37 +7,18 @@ stages:
 
 include:
   - project: nci-gdc/gitlab-templates
-    ref: 0.2.1
+    ref: 0.3.1
     file:
       - templates/global/full.yaml
       - templates/python/full.yaml
       - templates/common/python.yaml
 
-variables:
-  PRE_COMMIT_DOCKER_IMAGE: ${BASE_CONTAINER_REGISTRY}/python3.8-builder
-  PRE_COMMIT_VERSION: 3.2.1
-
-.prepareTox:
-  script:
-    - !reference [.load_github_key, script]
-    - apt-get update && apt-get install -y gnupg
-    - echo "deb http://apt.postgresql.org/pub/repos/apt/ focal-pgdg main 13" | tee
-      /etc/apt/sources.list.d/pgdg.list
-    - apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 7FCC7D46ACCC4CF8
-    - apt-get update
-    - apt-get install -y postgresql-client-13
-    - pip install "tox<4"
-  needs: [pre-commit]
-
 tox:
   stage: test
   parallel:
     matrix:
-      - PY_VER: ['3.6', '3.7', '3.8']
+      - BUILD_PY_VERSION: [python3.6, python3.7, python3.8, python3.9]
```

Comment:
Who is the 3.6 consumer?

Reply:
I will remove python3.6 in my next ticket.